packages/vue/src/components/atoms/SfInput/SfInput.vue

```diff
@@ -160,7 +161,26 @@ export default {
     type: {
       immediate: true,
       handler: function(value) {
-        this.inputType = value;
+        const ua = navigator.userAgent.toLowerCase();
```

Comment:
@AdamPawlinski you can't to that :) it's broke SSR :) because navigator is a client API

Reply:
just FYI

Reply:
thanks :)